AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -37,7 +37,7 @@ struct ChooseSessionTypeView: View {
                 mainContent
                 .fullScreenCover(isPresented: $isPowerABLinkActive) {
                     CreatingSessionFlowRootView {
-                        PowerABView(creatingSessionFlowContinues: $isPowerABLinkActive, sdSyncContinues: .constant(false), urlProvider: viewModel.passURLProvider)
+                        PowerABView(creatingSessionFlowContinues: $isPowerABLinkActive, urlProvider: viewModel.passURLProvider)
```

Comment:
Since creatingSessionFlowContinues is used also in syncing flow maybe it should have more generic name? Like "viewsFlowContinues"? or "wizzardViewsContinue"? I'm not very happy with those names either, but keeping it as "creatingSessionFlowContinues" is a bit misleading probably

Reply:
However, we can think about the name some other time